tasks/pi-hole.yml

```diff
@@ -6,6 +6,16 @@
     mode: 0755
   become: false
 
+- name: Get CIDR notation of subnet
+  set_fact:
+    mask_cidr: "{{ ip | ipaddr('prefix') }}"
+  vars:
+    ip: "{{ ansible_facts['default_ipv4']['address'] }}/{{ ansible_facts['default_ipv4']['netmask'] }}"
+
+- name: CIDR subnet is
+  ansible.builtin.debug:
+    msg: 'Subnet: {{ mask_cidr }}'
```

Comment:
Are these two debug tasks needed? Or were they just for testing/debug?

Reply:
The debug message isn't needed. I've removed that, however `mask_cidr` is used in the docker compose file